lib/pages/home/about_us/about_us_page.dart

```diff
@@ -66,38 +65,48 @@ class AboutUsPage extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     return Scaffold(
-        backgroundColor: Colors.white,
-        appBar: fBuildAppBar("assets/images/about_us_top_image.png"),
-        body: SingleChildScrollView(
-          child: Container(
-            padding: EdgeInsets.all(10),
-            child: Column(
-              crossAxisAlignment: CrossAxisAlignment.start,
-              children: <Widget>[
-                Padding(padding: EdgeInsets.only(top: 10)),
-                fPrintHeadingText(" O 4Business"),
-                Padding(padding: EdgeInsets.only(top: 10)),
-                fBuildAbout4BusinessText(),
-                Padding(padding: EdgeInsets.only(top: 10)),
-                Center(
-                  child: Image(
-                    alignment: Alignment.center,
-                    image: AssetImage("assets/images/logo_white.png"),
-                    width: 150,
-                  ),
-                ),
-                Padding(padding: EdgeInsets.only(top: 10)),
-                fPrintBoldJustifiedText(mAbout4BusinessTeamPart1),
-                Padding(padding: EdgeInsets.only(top: 5)),
-                fBuildOptions(),
-                fPrintJustifiedText(mAbout4BusinessTeamPart2),
-                Padding(padding: EdgeInsets.only(top: 10)),
-                fBuildButton(() => fChangePage(context, OfferPage.Id),
-                    "Sprawdź naszą ofertę"),
-                Padding(padding: EdgeInsets.only(top: 10)),
-              ],
+      body: CustomScrollView(
+        slivers: <Widget>[
```

Comment:
Całe te Widgety z slivers można by do jednej funkcji zapakować

Reply:
i lajk it